db/migrate/20240520191416_drop_accredited_individuals_accredited_organizations.rb

```diff
@@ -0,0 +1,5 @@
+class DropAccreditedIndividualsAccreditedOrganizations < ActiveRecord::Migration[7.1]
+  def change
+    drop_table :accredited_individuals_accredited_organizations
```

Comment:
```suggestion
    drop_table :accredited_individuals_accredited_organizations, if_exists: true
```

Could we add the conditional here? We sometimes run into problems in prod and making this idempotent would be nice.